sql/_05_plcsql/_01_testspec/_04_expression/_15_arithmetic_binary_op/_09_normal_int_mod_div_type_conversion/cases/04_15_09-02_int_arithmetic_op_boolean.sql

```diff
@@ -0,0 +1,1953 @@
+    left_string_datetime           string        := '2024-03-01 22:15:25'; 
+    left_string_timestamp          string        := '2024-05-05 23:30:45';
+    left_short                     short         := 8934;
+    left_int                       int           := 78234;
+    left_bigint                    bigint        := 278234;
+    left_numeric                   numeric(8,4)  := 5678.123;
+    left_float                     float         := 3677.345;
+    left_double                    double        := 56789.1234;
+    left_date                      date          := DATE'2024-03-05';
+    left_time                      time          := TIME'23:59:59';
+    left_datetime                  datetime      := DATETIME'2024-03-01 22:15:25';
+    left_timestamp                 timestamp     := TIMESTAMP'2024-05-05 23:30:45';
+    left_sys_refcursor             sys_refcursor ;
+    cursor  left_cursor is select * from dual ;
+    
+    right_boolean                 boolean       := FALSE;
+    right_string                  string        := '999';
+    right_string_null             string        := NULL; 
+    right_string_boolean        string        := 'FALSE'; 
+    right_string_short            string        := '34';                               
+    right_string_int              string        := '234';                              
+    right_string_bigint           string        := '1234';                             
+    right_string_numeric          string        := '1234.567';                         
+    right_string_float            string        := '432.321';                        
+    right_string_double           string        := '1234.56789';                 
+    right_string_date             string        := '2002-03-05';                 
+    right_string_time             string        := '03:59:59';               
+    right_string_datetime         string        := '2002-02-02 02:22:22';
+    right_string_timestamp        string        := '2002-01-01 01:01:01';
+    right_short                   short         := 34;
+    right_int                     int           := 234;
+    right_bigint                  bigint        := 1234;
+    right_numeric                 numeric(8,4)  := 1234.567;
+    right_float                   float         := 432.321;
+    right_double                  double        := 1234.56789;
+    right_date                    date          := DATE'2002-03-05';
+    right_time                    time          := TIME'03:59:59';
+    right_datetime                datetime      := DATETIME'2002-02-02 02:22:22';
+    right_timestamp               timestamp     := TIMESTAMP'2002-01-01 01:01:01';
+    right_sys_refcursor           sys_refcursor ;
+    cursor  right_cursor is select * from dual ;
+begin 
+
+    dbms_output.put_line('***************************************');
+    dbms_output.put_line('null = ' ||  NVL(null,'NULL') ); 
+    dbms_output.put_line('right_boolean = ' ||'FALSE','NULL') ); 

```

Comment:
```suggestion
    dbms_output.put_line('right_boolean = ' ||'FALSE'); 

```
Same for all procedures below.